### DISCUSSION TITLE
NPC Model Swap

### DISCUSSION THREAD

**Discussion by Jinxer42** [runelite/runelite#17036] [Ideas]
I would love a plugin that allows one to replace a NPC model with another NPC model.  Ideally (but not essentially) the model would have the appropriate walking animation.  The plugin can have the restriction that a model replacement has to be for models of the same size. I think this would be a great addition to the Hub.

For me personally this plugin would be very helpful.  I currently am stuck on the Eyes of Glouphrie quest.  I cannot bring myself to kill the little “evil” black creatures because I find them to be the most adorable things in Gielinor.  As such I want to be able to swap their model with gnomes and make gnomes look like evil creatures.  This way I can finish the quest by slaughtering the gnomes and keeping the evil creatures safe.

Sincerely,
Someone who really wants to finish Eyes of Glouphrie

**Comment by catbeeef** [comment]
would like this for phobia reasons x_x

**Reply by cdfisher** [reply]
You can probably use "Entity Masker" from the Plugin Hub.

**Reply by catbeeef** [reply]
amazing, thank you so much

**Comment by capSAR273** [comment]
I'd love this so I can put the santa hats/party hats on the NPCs and pets that get changed during Christmas, its cute but being limited to Christmas :(

**Comment by greyapple2** [comment]
Grow up and just beat the quest.  When you beat it you won't feel bad about it anyway.  They're not cute, they're monsters.  Now grow a pair.

**Reply by catbeeef** [reply]
are you a professional asshole, or is that just a hobby?

**Reply by Jinxer42** [reply]
> Grow up and just beat the quest. When you beat it you won't feel bad about it anyway. They're not cute, they're monsters. Now grow a pair.

I just made a plugin that did what I wanted: https://runelite.net/plugin-hub/show/save-evil-creatures.  Maybe one day I will need to "grow up" or "grow a pair", but today is not that day! 😄

**Reply by greyapple2** [reply]
> are you a professional asshole, or is that just a hobby?

It's a hobby bbygrl
